### ISSUE TITLE
Migration from 1.5 to 2.x

### ISSUE THREAD

**Issue by vasyugan** [jonasoreland/runnerup#945]
I just installed the current 2.x release from F-droid, before that I had version 1.5 from Google play installed. There is no obvious way of importing history and settings from the old version.  Not a matter of life and death, but this would definitely be nice to have. I thought I could just back the old version up with Titanium, uninstall it, install the newer release and restore the app data, but aparently, the versions in GPlay and F-Droid have different identifiers.

**Comment by gerhardol** [comment]
The apps can be installed in parallel.
Export the database in the previous version, this will put the file runnerup.db.export in the sd card root, it can be moved to other devices too.
(You may have to allow file access in settings.)
Then import in the new app.

The Play version has not been updated much, but I have access now and the beta in Play is the same version as in F-Droid.
There are a few more features in Play version, but F-Droid is just free software.

**Comment by vasyugan** [comment]
Thanks! It was a pleasent surprise to see that this great app is not abandoned (the stable version had not been updated in a long time). This is probably not the correct place for asking, but what's the deal with the "runner up live" account? Is this a planned feature?

**Comment by gerhardol** [comment]
RunnerUp Live was a companion live server developed separately, but it was abandoned itit seems even before I found the app. See #351 for info.

I spend quite some time just to follow changes Google requires, so I have not much time to add new features myself, but more contributors are welcome. Not abandoned.
